dotcom-rendering/src/components/FeatureCard.stories.tsx

```diff
@@ -20,7 +20,7 @@ const cardProps: CardProps = {
 	},
 	imagePositionOnDesktop: 'top',
 	isExternalLink: false,
-	isPlayableMediaCard: false,
+	canPlayInline: true,
```

Comment:
It's just a story so not super important, but was this meant to be changed to true?

Reply:
yep, feature cards should play inline 👍